test/integration/react-browser/src/App.tsx

```diff
@@ -1,6 +1,5 @@
 import React, {useEffect} from 'react';
 import {Chart, DoughnutController, ArcElement} from 'chart.js';
-import {merge} from 'chart.js/helpers';
```

Comment:
This was intentional since this broke in 3.9 so we need an import from the helpers. What we can do instead is import PI or TAU and use that as a value for the dataset

Reply:
restored. `PI` and `TAU` don't seem to be available. perhaps they should be, but i'd rather handle that as a separate issue. I just used `merge`